STORIES.md

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
I like that you made an readme for the user stories. It shows you were thinking step by step and referring to what the 'customer' wanted